clients/python/tests/conftest.py

```
@@ -317,3 +339,85 @@
+
+    os.environ["API_HOST"] = REGISTRY_URL
+    config = schemathesis.config.SchemathesisConfig.from_path(f"{pytestconfig.rootpath}/schemathesis.toml")
+    local_schema_path = f"{pytestconfig.rootpath}/../../api/openapi/model-registry.yaml"
+    schema = schemathesis.openapi.from_path(
+        path=local_schema_path,
+        config=config,
+    )
+    schema.config.output.sanitization.update(enabled=False)
+    return schema
+
+@pytest.fixture
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)